packages/core/lib/src/widgets/twitter_logo.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+/// Widget for the Twitter Logo
+class TwitterLogo extends StatelessWidget {
+  /// Creates widget for the Twitter Logo
+  const TwitterLogo({
+    super.key,
+    this.size = 36,
+  });
+
+  /// Width and Height of the logo
+  final double size;
+
+  @override
+  Widget build(BuildContext context) {
+    return Image.asset(
+      'assets/images/twitter-logo.png',
+      package: 'core',
+      width: size,
+      height: size,
+    );
+  }
+}
```

Comment:
Do we have a Widgetbook use-case for this?

Reply:
Nope. Will add one.